public/docs/_examples/quickstart/ts/app/main.ts

```diff
@@ -1,9 +1,6 @@
-// #docregion
-// #docregion import
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 
 import { AppModule } from './app.module';
-// #enddocregion import
 
 const platform = platformBrowserDynamic();
 platform.bootstrapModule(AppModule);
```

Comment:
Two votes to remove `export` on component class to my one;  you both win.
Fixed `platformBrowserDynamic().boot..`